sql/sql_parse.cc

```diff
@@ -2572,7 +2572,7 @@ static bool lock_tables_for_backup(THD *thd)
   @return FALSE in case of success, TRUE in case of error.
 */
 
-static bool lock_binlog_for_backup(THD *thd)
+bool lock_binlog_for_backup(THD *thd)
```

Comment:
This changes are quite obvious. I did it to make this function accessible from tokudb-hotbackup plugin. I don't think we need blueprints for such small changes. At least it was not required for this pull request https://github.com/percona/percona-server/pull/1270 , where I did the same thing for global variable in innodb. @laurynas-biveinis, what do you think?